spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -161,4 +162,103 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    let(:title) { Faker::Book.title }
+    let(:type) { :review }
+    let(:content) { Faker::Lorem.paragraphs(number: 3).join("\n") }
+    let!(:note_count) { Note.count }
+
+    context 'when there is a user logged in' do
+      let(:params) { { title: title, type: type, content: content } }
+
+      include_context 'with authenticated user'
+
+      context 'when creating a valid note' do
+        before { post :create, params: params }
+
+        it 'responds with 201 status' do
+          expect(response).to have_http_status :created
+        end
+
+        it 'responds with the expected message' do
+          expect(response_body['message']).to eq I18n.t('responses.note.created')
+        end
+
+        it 'creates a note' do
+          expect(Note.count).to eq(note_count + 1)
+        end
+
+        it 'associates the user' do
+          expect(Note.last.user.id).to eq(user.id)
+        end
+      end
+
+      context 'when a required parameter is missing' do
+        let(%i[title type content].sample) { nil }
+
+        before { post :create, params: params }
+
+        it 'responds with 400 status' do
+          expect(response).to have_http_status :bad_request
+        end
+
+        it 'responds with the expected error' do
+          expect(response_body['error']).to eq I18n.t('responses.global.missing_required_params')
+        end
+
+        it 'does not create a note' do
+          expect(Note.count).to eq(note_count)
+        end
```

Comment:
yy así como hay un to_change hay un not to change 🚀 

Podemos investigarlo un poquito 🤗